app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/index.tsx

```diff
@@ -10,11 +12,16 @@ import type { UseStubImagesInfoResult } from './hooks/useStubImagesInfo'
 
 export function ImageGalleryContainer(): JSX.Element {
   const imagesInfo = useStubImagesInfo()
-
+  const { t } = useTranslation('run_details')
+  const { isCameraEnabled } = useCameraUsageSettings()
   return (
     <div className={styles.gallery_container}>
       <GalleryHeader imagesCount={imagesInfo.length} />
-      <GalleryContent imagesInfo={imagesInfo} />
+      {isCameraEnabled ? (
+        <GalleryContent imagesInfo={imagesInfo} />
+      ) : (
+        <InfoScreen content={t('no_images_available')} />
+      )}
```

Comment:
```suggestion
        <GalleryContent imagesInfo={imagesInfo} />
```
Whether or not we render the `InfoScreen` should be conditional on whether images exist for this `runId`, so we can't quite do [EXEC-1977](https://opentrons.atlassian.net/browse/EXEC-1977) yet (sorry, I'm close to unblocking you on this).